presto-product-tests/src/main/java/io/prestosql/tests/hive/TransactionalTestHelper.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.tests.hive;
+
+import com.google.common.net.HostAndPort;
+import com.google.inject.Inject;
+import com.google.inject.name.Named;
+import io.prestosql.plugin.hive.metastore.thrift.NoHiveMetastoreAuthentication;
+import io.prestosql.plugin.hive.metastore.thrift.ThriftHiveMetastoreClient;
+import io.prestosql.plugin.hive.metastore.thrift.Transport;
+import org.apache.thrift.TException;
+
+import java.net.URI;
+import java.util.Optional;
+
+public final class TransactionalTestHelper
```

Comment:
As it is used in just a single place now please inline into `TestHiveTransactionalTable.java`

Reply:
It is getting used in https://github.com/prestosql/presto/pull/5404/files tests as well, so I thought of keeping the utils separately, to keep the test file shorter. thoughts?

Reply:
Then please rename the file to `TestHiveMetastoreClientFactory`